crates/typst/src/layout/transform.rs

```diff
@@ -335,6 +335,115 @@ cast! {
     length: Length => ScaleAmount::Length(length),
 }
 
+/// Skews content.
+///
+/// Skews an element in horizontal and/or vertical direction. The layout will
+/// act as if the element was not skewed unless you specify `{reflow: true}`.
+///
+/// # Example
+/// ```example
+/// #skew(-12deg)[This is some fake italic text.]
+/// ```
+#[elem(Show)]
+pub struct SkewElem {
+    /// The horizontal skewing angle.
+    ///
+    /// ```example
+    /// #skew(ax: 30deg)[Skewed]
+    /// ```
+    ///
+    #[parse(args.named("ax")?.or(args.find()?))]
+    #[default(Angle::zero())]
+    pub ax: Angle,
+
+    /// The vertical skewing angle.
+    ///
+    /// ```example
+    /// #skew(ay: 30deg)[Skewed]
+    /// ```
+    ///
+    #[parse(args.named("ay")?.or(args.find()?))]
+    #[default(Angle::zero())]
+    pub ay: Angle,
+
+    /// The origin of the skew transformation.
+    ///
+    /// The origin will stay fixed during the operation.
+    ///
+    /// ```example
+    /// #set text(spacing: 16pt)
```

Comment:
I think this example is too complex for the docs. The image it generates is also not really instructive. Maybe we can find something simpler. Also, please use `kebab-case`.

Reply:
Yes, I see this. My thought here was to show both the original and the skewed square to illustrate where the fixed point is but I see that this is too cumbersome for the docs. 

How about this simpler example where still some unchanged characters are shown for comparison:

```typst
X #box(skew(ax: -30deg, origin: center + horizon)[X]) X \
X #box(skew(ax: -30deg, origin: bottom + left)[X]) X \
X #box(skew(ax: -30deg, origin: top + right)[X]) X
```